src/components/EventForm.js

```diff
@@ -6,21 +6,25 @@ import moment from "moment";
 import uuid from "uuid";
 
 export default class EventForm extends React.Component {
-  state = {
-    title: "",
-    description: "",
-    organisator: "",
-    city: "",
-    category: "",
-    image: false,
-    id: uuid(),
-    createdAt: moment(),
-    startDate: moment(),
-    startDateId: "",
-    endDate: moment(),
-    endDateId: "",
-    calendarFocused: null,
-  };
+  constructor(props) {
```

Comment:
You don't need constructor if you only want to initialize state there. And yes, you have access to props outside of constructor by `this.props`.